docs/src/starknet/tx-status.md

```diff
@@ -28,4 +28,29 @@ Success: Transaction status retrieved
 Finality Status:  Accepted on L1
 Execution Status: Succeeded
 ```
+
 </details>
+
+## Block Explorer Links
```

Comment:
Explorer links are used also in commands other than `tx-status` - wdyt about moving this to a separate docs section?